source/docs/hardware/mechanical/linear-bushings.rst

```diff
@@ -0,0 +1,109 @@
+
+.. image:: images/linear-bushings/straight-vs-flanged.png
+
+Straight (**Photo 1**) and flanged bushings (**Photo 2**) share a similar structural design. The main advantage of using flanged Linear Bushing lies in its compact design (**Figure 1**) as the integrated structure saves space. The outer cylinder flange allows direct mounting of the bushing and allows flanged bushings to maintain higher load capacity than standard Linear Bushings. A hardened, precision flanged outer cylinder (housing), made out of chrome steel or corrosion resistant steel, is advanced in its quality, and lower in cost compared to “self-made” flanged housing.
+
+.. image:: images/linear-bushings/flanged-bushing.png
+
+Choosing Between a Straight and Flanged Linear Bushing
+------------------------------------------------------
+
+The following factors should be considered when making Linear Bushing selections.
+
+- Decide whether force will be applied to the Linear Bushing. Choose a flanged type if the Linear Bushing must bear force.
+- Decide how much space is available on the surface to which the Linear Bushing is to be attached. (Refer to part 3)
+
+The Linear Bushing in the component a) receives inertia force from the moving component, therefore the Linear Bushing must be firmly screwed to the housing. As for component b), an Air Cylinder moves the Shaft in the Linear Bushing. The `Retaining Ring <http://us.misumi-ec.com/vona2/mech/M1800000000/M1809000000/?searchFlow=results2category?utm_source=FIRST>`_ fixing the Linear Bushing only receives the frictional force from the Shaft. Therefore, a compact design using a straight type is fine. The same can be said for c).
+
+As shown in Figure 2, depending on the design, Linear Bushings can either move while Shafts are fixed, or be stationary (fixed) while Shafts are in motion.
+
+
+- X-axis: If the Linear Bushing moves, use the Flanged type bushings.
+- Y & Z-axis: For fixed Linear Bushing, use Straight type with Retaining Ring or `Stopper Plate <http://us.misumi-ec.com/vona2/mech/M0300000000/M0311000000/M0311020000/?searchFlow=results2category?utm_source=FIRST>`_.
+
+How to Install Linear Bushings
+------------------------------
+
+To fix a Straight Linear Bushing, using Retaining Ring or Stopper Plate (Fixing Plate as shown in photo 3) see
+
+.. image:: images/linear-bushings/installing-bushing.png
+
+.. note:: Load rating of Linear Bushing varies according to the load position on the circumference. Linear Bushing, usually has 4-6 rows/ball tracks that are set on even angle. When installing, if possible, avoid positioning Linear Bushing so that the ball track is under direct load (Figure 4), otherwise that row will directly bear the load (Figure 4a).
+
+   For example, Figure 4 shows a Linear Bushing with 5 rows. The variance of dynamic load rating is as follows:
+
+      right figure ÷ left figure.
```

Comment:
Try to use latex where math is involved. It adds emphasis and just generally looks nicer